Example/Podfile.lock

```diff
@@ -18,17 +18,17 @@ PODS:
     - Extraction (>= 1.2.1)
     - Folly (= 2016.09.26.00)
     - glog (= 0.3.4)
-    - React/Core (= 0.54.4)
-    - React/CxxBridge (= 0.54.4)
-    - React/RCTAnimation (= 0.54.4)
-    - React/RCTCameraRoll (= 0.54.4)
-    - React/RCTImage (= 0.54.4)
-    - React/RCTLinkingIOS (= 0.54.4)
-    - React/RCTNetwork (= 0.54.4)
-    - React/RCTText (= 0.54.4)
+    - React/Core (= 0.55.4)
+    - React/CxxBridge (= 0.55.4)
+    - React/RCTAnimation (= 0.55.4)
+    - React/RCTCameraRoll (= 0.55.4)
+    - React/RCTImage (= 0.55.4)
+    - React/RCTLinkingIOS (= 0.55.4)
+    - React/RCTNetwork (= 0.55.4)
+    - React/RCTText (= 0.55.4)
     - SDWebImage (< 4, >= 3.7.2)
-    - SentryReactNative (= 0.30.3)
-    - yoga (= 0.54.4.React)
+    - SentryReactNative (= 0.38.0)
+    - yoga (= 0.55.4.React)
```

Comment:
they don't bump the number until the release

and I updated sentry-reactnative also